src/lib/components/composed/list/Paginator.css

```diff
@@ -1,4 +1,13 @@
 :root {
-  --paginator-default-selected-background-color: rgb(var(--global-color-primary-500));
-  --paginator-default-selected-color: rgb(var(--global-color-grey-100))
+  --paginator-page-button-default-selected-background-color: rgb(var(--global-color-primary-500));
```

Comment:
Fose non mi sono spiegato bene, scusami.
La convezione che usiamo rispetta le seguenti regole:

1. Tutte le variabili CSS che vengono utilizzate dentro un componente iniziano con il nome del componente. Quindi in questo caso è tutto corretto, perchè tutte le variabili iniziano con `paginator`
2. Le varibili di default iniziano sempre con il nome del componente (in questo caso paginator) e subito dopo hanno la stringa `default`. Quindi in questo caso un esempio potrebbe essere `paginator-default-page-button-selected-color`. Oppure `paginator-default-page-button-selected-padding`.